### DISCUSSION TITLE
Numerical data on ogbl_wikidata2

### DISCUSSION THREAD

**Discussion by aahad5555** [facebookresearch/ssl-relation-prediction#12] [ogbl-wikikg2] [answered]
Hi, I am quite new to knowledge graphs and came across ogbl_wikidata2 dataset. I cant get my head around that this dataset is just numbers. I have a simple question why is that?and is their any real entity or data behind that?

**Comment by yihong-chen** [comment, marked as answer]
Hi @aahad5555, I guess you mean `ogbl-wikikg2`. If you download the data by using the [OGB dataloader](https://ogb.stanford.edu/docs/linkprop/#ogbl-wikikg2),  you will find the mapping file under the folder `dataset/ogbl_wikikgv2/mapping`, which map numbers to wikidata entity/relation IDs. 

In a IPython shell, running the following scripts should show you the folder
```python
from ogb.linkproppred import LinkPropPredDataset

dataset = LinkPropPredDataset(name = 'ogbl-wikikg2')

!ls ./dataset/ogbl_wikikg2/mapping
```

**Reply by aahad5555** [reply]
Thanks and greatly appreciated.
